pkg/controller/datavolume/controller-base.go

```diff
@@ -291,6 +304,7 @@ func addDataVolumeControllerCommonWatches(mgr manager.Manager, dataVolumeControl
 		}
 	}
 
+	// FIXME: Consider removing this Watch. Not sure it's needed anymore as PVCs are Pending in this case.
```

Comment:
So this watch doesn't solve the original problem anymore? Does something else solve it now?
(since this type of DVs is always "Pending")

EDIT:
This does, right?
```With this feature enabled, you no longer need to create a default StorageClass first and then a PVC to assign the class. Instead, any PVCs without a StorageClass assigned will now be retroactively updated to include the default StorageClass```
https://kubernetes.io/blog/2023/08/18/retroactive-default-storage-class-ga/

Reply:
Right, but I think we may remove the Watch in a separate PR.